Mention default CA cert rotation in the remote clusters docs (#3415)
elastic/cloud-on-k8s: docs/orchestrating-elastic-stack-applications/elasticsearch/remote-clusters.asciidoc

NOTE: Beware of copying the source Secret as-is into a different namespace. See <<{p}-common-problems-owner-refs, Common Problems: Owner References>> for more information.

> NOTE: CA certificates are automatically rotated after one year by default. You can link:k8s-operator-config.html[configure] this period. Make sure to keep up-to-date the copy of the certificates secret.

Reviewer: ```suggestion
NOTE: CA certificates are automatically rotated after one year by default. You can link:k8s-operator-config.html[configure] this period. Make sure to keep the copy of the certificates Secret up-to-date.
```

> NOTE: CA certificates are automatically rotated after one year by default. This period is link:k8s-operator-config.html[configurable]. You should ensure the certificates secret copy is maintained up-to-date.

Reviewer 2: ```suggestion
NOTE: CA certificates are automatically rotated after one year by default. You can link:k8s-operator-config.html[configure] this period. Make sure to keep up-to-date the copy of the certificates secret.
```
